Dockerfile-py2.7

```diff
@@ -12,8 +15,12 @@ RUN apt-get update && \
         python-pyqt5 \
         python-pyside \
         python-pyside2 \
+        pyside2-tools \
         xvfb
 
+# Make pyside2uic availble for Python 2.x
+RUN cp -avr /usr/lib/python3/dist-packages/pyside2uic /usr/local/lib/python2.7/dist-packages
```

Comment:
Okay, well that's fine. This solution should work, they're Python 2 compatible at least?